server/game/cards/01-DTR/CheatinVarmint.js

```diff
@@ -0,0 +1,31 @@
+const ActionCard = require('../../actioncard.js');
+
+class CheatinVarmint extends ActionCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Resolution: Lower hand by 2 ranks',
+            playType: ['resolution'],
+            choosePlayer: true,
+            cost: ability.costs.payGhostRock(5),
+            message: context =>
```

Comment:
The hand rank modification is exception from the rule that we should use `message` property. That is because message property writes before the rank is manipulated, so the "Current rank is {}" will show rank before it is manipulated. 
Therefore cards doing card manipulations should write messages to log in `handler` after the manipulation.

Reply:
Ah, that makes sense. I should have caught that.